### PR TITLE
Platform scan follow-ups

### PR DESCRIPTION
**Pull request by mrogers950**
- Use new SA for api-resource-collector.
- Use golang-1.13 builder for api-resource-collector image.
- Remove personal repo use.
- Don't use 777 for staging directories.

(note, I pushed some extra images to quay.io/compliance-operator to get rid of my own repos: openscap-ocp:1.3.3 and ocp4-openscap-content:platform_test)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,8 +17,8 @@ IMAGE_REPO?=quay.io/compliance-operator
 RUNTIME?=podman
 
 # Temporary
-OPENSCAP_IMAGE_REPO?=quay.io/mrogers950
-OPENSCAP_IMAGE_TAG?=yamlprobe
+OPENSCAP_IMAGE_REPO?=quay.io/compliance-operator
```
Comment (JAORMX) [inline Makefile:20]: you could delete this one and use $(IMAGE_REPO) which is in line 16.

**Comment by jhrozek** [comment]
FWIW these fixups address the concerns I had with the original PR

**Comment by mrogers950** [comment]
Lots of degraded cluster events
/retest

**Comment by mrogers950** [comment]
The quay images I pushed were set to private repos for some reason.. I set them to public, trying again.
/test e2e-aws

**Comment by mrogers950** [comment]
```
 level=info msg="Consuming OpenShift Install (Manifests) from target directory"
level=info msg="Consuming Common Manifests from target directory"
level=info msg="Consuming Openshift Manifests from target directory"
level=info msg="Credentials loaded from the \"default\" profile in file \"/etc/openshift-installer/.awscred\""
level=info msg="Creating infrastructure resources..."
level=error
level=error msg="Error: error updating EC2 Route Table (rtb-0f04320f731a9dd51) tags: error tagging resource (rtb-0f04320f731a9dd51): InvalidRouteTableID.NotFound: The routeTable ID 'rtb-0f04320f731a9dd51' does not exist"
level=error msg="\tstatus code: 400, request id: eeb2c9d1-189d-4439-b0e2-201e4751970a"
level=error
level=error msg="  on ../tmp/openshift-install-733978875/vpc/vpc-private.tf line 1, in resource \"aws_route_table\" \"private_routes\":"
level=error msg="   1: resource \"aws_route_table\" \"private_routes\" {"
level=error
level=error
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform" 
```
/retest

**Comment by mrogers950** [comment]
Fixed an issue with the rolebindings: The api-resource-collector SA needed to keep the resultcollector permissions in addition to cluster-reader. This should be good to merge now.

**Pull request merged**